### PR TITLE
Misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3`

### CODE REVIEW

File: m4/bswap.m4 [hunk 1]
```diff
@@ -36,7 +36,7 @@ dnl Dtermine whether the compiler has the __builtin_bswap32() intrinsic which
 dnl is likely to be present for most versions of GCC as well as Clang.
 
 AC_DEFUN([XIPH_C_BSWAP32],
-[AC_CACHE_CHECK(for bswap32 instrinsic,
+[AC_CACHE_CHECK(for bswap32 intrinsic,
```
Comment (luzpaz) [inline m4/bswap.m4]: please review

File: m4/bswap.m4 [hunk 2]
```diff
@@ -63,7 +63,7 @@ dnl Dtermine whether the compiler has the __builtin_bswap16() intrinsic which
 dnl is likely to be present for most versions of GCC as well as Clang.
 
 AC_DEFUN([XIPH_C_BSWAP16],
-[AC_CACHE_CHECK(for bswap16 instrinsic,
+[AC_CACHE_CHECK(for bswap16 intrinsic,
```
Comment (luzpaz) [inline m4/bswap.m4]: please review

File: src/plugin_xmms/configure.c [hunk 3]
```diff
@@ -46,7 +46,7 @@
 #include "locale_hack.h"
 
 /*
- * Initialize Global Valueable
+ * Initialize Global Valuable
```
Comment (luzpaz) [inline src/plugin_xmms/configure.c]: should this actually be `Variable` ?

Comment (erikd) [reply src/plugin_xmms/configure.c]: Can you please change that the "variable"?

**Pull request closed without merging**